lib/app/routes/static.rb

```diff
@@ -65,6 +65,11 @@ class Static < Core
         status 404
         erb :"errors/not_found"
       end
+
+      get '/faq' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.